lib/iex/lib/iex/h_elixir.ex

```diff
@@ -0,0 +1,143 @@
+defmodule IEx.H_Elixir do
```

Comment:
The former option seems natural to me. Not sure about `ErlangStub` though, maybe just `Erlang` even is it's a stub module.